src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/PrisonOffenderEventListener.kt

```diff
@@ -38,6 +47,10 @@ class PrisonOffenderEventListener(
   }
 }
 
+enum class Reason {
+  TRANSFERRED, // TODO need to confirm other valid reasons, is the prison Id the one going to or from and so on.
```

Comment:
- nomsNumber string - NOMIS offender number
- reason enum - reason for the receive. Will be one of these values below. It has been observed that administration mistakes in NOMIS can cause the reason for RECALL and CONVICTED to be not always be accurate.
   -- **NEW_ADMISSION** prisoner is entering prison due to new charges
   -- **READMISSION** prisoner is re-entering prison due to a new court order or recall
   -- **TEMPORARY_ABSENCE_RETURN** prisoner is returning from temporary absence e.g. day release
   -- **RETURN_FROM_COURT** prisoner is returning from court
   -- **TRANSFERRED** prisoner has been transferred from another prison
- detail string - further human readable information about the reason. The contents is developer focused and not to be relied on. ExampleRecall referral date 2021-06-13
- prisonId string - the current prison where the prisoner is located

Reply:
Prison ID is where they are located.  Be best to handle NEW_ADMISSION, READMISSION, TRANSFERRED